.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 package-lock.json
```

Comment:
Package-lock should always be committed to the repository as it acts like a source of truth for project dependencies